pkg/logger/logger.go

```diff
@@ -105,22 +143,32 @@ func (l *Logger) log(logColor *color.Color, message string) {
 
 func (l *Logger) SetLogLevel(logLevel LogLevel) error {
 	l.LogLevel = logLevel
+
+	if l.StyledLogger != nil {
+		switch logLevel {
+		case LogLevelTrace:
+			l.StyledLogger.SetLevel(AtmosTraceLevel)
+		case LogLevelDebug:
+			l.StyledLogger.SetLevel(log.DebugLevel)
+		case LogLevelInfo:
+			l.StyledLogger.SetLevel(log.InfoLevel)
+		case LogLevelWarning:
+			l.StyledLogger.SetLevel(log.WarnLevel)
+		case LogLevelOff:
+			l.StyledLogger.SetLevel(log.ErrorLevel)
+		}
+	}
+
 	return nil
 }
 
 func (l *Logger) Error(err error) {
 	if err != nil && l.LogLevel != LogLevelOff {
```

Comment:
This condition doesn't make sense to me. Isn't output regulated by charmbracelet? I think we should get rid the `l.LogLevel` check.